### PR TITLE
fffuu: init at unstable-2018-05-26

### LINKED ISSUE #52468: Request to package fffuu (Verifying Iptable Rules)
https://github.com/diekmann/Iptables_Semantics/tree/master/haskell_tool

The ffuu tool analyses iptable rules and dumps them. I thought it would be pretty cool to try it out on NixOS.

### PR DESCRIPTION
**Pull request by marsam**
###### Motivation for this change
closes https://github.com/NixOS/nixpkgs/issues/52468

cc: @CMCDragonkai 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/fffuu/default.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{ mkDerivation, haskellPackages, fetchFromGitHub, lib }:
+
+mkDerivation rec {
+  pname = "fffuu";
+  version = "unstable-2018-05-26";
```
[the remark this answers is not in the document]
Comment (ryantm) [reply pkgs/tools/misc/fffuu/default.nix]: [Nixpkgs manual package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming):
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

**Comment by ryantm** [comment]
@GrahamcOfBorg build fffuu

**Comment by ryantm** [comment]
Build failure:
```
  Setup: Encountered missing dependencies:
  containers ==0.5.*
cannot build derivation '/nix/store/sgl0pfyfbrkn6irvsjnnfs2psns8v32y-env.drv': 1 dependencies couldn't be built
```

**Comment by marsam** [comment]
Fixed

**Comment by ryantm** [comment]
@GrahamcOfBorg build fffuu

**Comment by marsam** [comment]
disabled tests because fails in sandbox

**Comment by aanderse** [comment]
ping (triage)

**Pull request merged**